src/vue/text-editor/actions-block.vue

```diff
@@ -9,6 +9,10 @@
             @click="uploadTexts" :disabled="!docSourceEditAvailable" >
             {{ l10n.getMsgS('ACTIONS_UPLOAD_TITLE') }}
         </button>
+        <button class="alpheios-editor-button-tertiary alpheios-actions-button alpheios-actions-metadata" 
+            @click="toggleMetadata" :disabled = "!metadataAvailable">
+            {{ l10n.getMsgS('ACTIONS_METADATA_TITLE') }}

```

Comment:
I think we need a visual indicator of how to close the metadata section.  Maybe the button text needs to change if the metadata form is active or not, or the color should change?

Reply:
Let's do the same way as done for options - Show metadata / Hide metadata - it is biger , but it would be consistent.
And later while working on #86 I would think how to redesign menus.